### DISCUSSION TITLE
Add auto skip

### DISCUSSION THREAD

**Discussion by gamerturtlepoggers** [ppy/osu#13196] [Ideas]
**Describe the new feature:** 
There is an option to automatically skip intros
**Proposal designs of the feature:** 
For example if you click f10 is disables mouse buttons maybe there could be something like that to disable intros

**Comment by peppy** [comment]
I don't think this is something we want, but will leave it open as a proposal.

**Comment by szmarczak** [comment]
I approve this feature. It's really annoying to have to skip intros.

**Comment by bdach** [comment]
@szmarczak please stop adding "i agree" comments. they are noise. use reactions on the first post instead.

**Comment by szmarczak** [comment]
I don't consider `It's really annoying to have to skip intros.` as noise. The OP just proposed a feature without any explaination behind this. It's just frustrating it takes a little while to skip it manually, therefore you end up losing minutes (if not hours) in total.

I just wanted to express my opinion - but feel free to disagree. In that case I'll have no other choice than to skip manually :)

> use reactions on the first post instead.

Already did that.
